content/en/tracing/setup_overview/compatibility_requirements/java.md

```diff
@@ -45,6 +45,7 @@ Beta integrations are disabled by default but can be enabled individually:
 | Java Servlet Compatible | 2.3+, 3.0+ | Fully Supported | `servlet`, `servlet-2`, `servlet-3`            |
 | Jax-RS Annotations      | JSR311-API | Fully Supported | `jax-rs`, `jaxrs`, `jax-rs-annotations`, `jax-rs-filter` |
 | Jetty                   | 7.0-9.x    | Fully Supported | `jetty`                                        |
+| Mulesoft                | 4          | Fully Supported | `mule`                                         |
```

Comment:
Mule is not enabled by default, and is dependent on the Grizzly server and client instrumentations that are marked as `[Beta][2]`